pyvista/trame/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""Trame interface for PyVista."""
+from trame.widgets.vtk import VtkLocalView, VtkRemoteView
+
+
+class _BasePyVistaView:
+    def __init__(self, plotter):
+        self._plotter = plotter
+        self.pyvista_initialize()
+
+    def pyvista_initialize(self):
+        if not hasattr(self._plotter, 'ren_win'):
+            raise RuntimeError(
+                'The render window for this plotter has been destroyed. Do not call `show()` for the plotter before passing to trame.'
+            )
+        if not self._plotter.camera_set:
+            self._plotter.view_isometric()
+
+
+class PyVistaRemoteView(VtkRemoteView, _BasePyVistaView):
+    """PyVista wrapping of trame VtkRemoteView."""
+
+    def __init__(self, plotter, interactive_ratio=1, ref='view', **kwargs):
+        """Create a trame remote view from a PyVista Plotter."""
+        _BasePyVistaView.__init__(self, plotter)
```

Comment:
Since multiple inheritance seems like a given, it might be prudent to use `super()` in these examples to show best practices. This might include calling `super().__init__(self)` in `_BasePyvistaView.__init__()`. (Of course the lack of kwargs already decreases flexibility in cooperation, but still.)

Reply:
yeah... I'm always not sure how best to handle multi-inheritence like this and didn't see a clean/clear way to do this with `super()`. I'll come back to this with a fresh mind soon here and try to clean it up

Reply:
I don't think there is a way to do this with `super()`